EIPS/eip-is-contract.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
```suggestion
---
eip: 7761
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.